### DISCUSSION TITLE
Why can't we have Flyout and bottom Tabs together in Shell?

### DISCUSSION THREAD

**Discussion by siddharthsingh89** [dotnet/maui#8598] [General]
As per the [documentation](https://docs.microsoft.com/en-us/dotnet/maui/fundamentals/shell/tabs), the top level of navigation in a Shell app is **either** a flyout or a bottom tab bar, depending on the navigation requirements of the app. 
Also, The TabBar type **disables** the flyout.

Is there any specific reason for this? It would be great if we can have both together.

Apps like Outlook mobile, Twitter, LinkedIn, Gmail etc. have both Flyout and Bottom Tab navigation.

**Comment by angelru** [comment]
@siddharthsingh89  any solution?

**Comment by rpharon** [comment]
Has anyone came accross to achieving having flyout and tabbar at the same time?

**Reply by JRosanowski** [reply]
Shell Flyout with Sharpnado.Tabs might work for you?
https://github.com/roubachof/Sharpnado.Tabs

**Reply by dartasen** [reply]
Shell Flyout with devexpress TabView also worked for me https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.TabView
